packages/webview_flutter/webview_flutter_platform_interface/lib/src/v4/webview_widget_delegate.dart

```diff
@@ -0,0 +1,90 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/gestures.dart';
+import 'package:flutter/widgets.dart';
+import 'package:plugin_platform_interface/plugin_platform_interface.dart';
+import 'package:webview_flutter_platform_interface/src/v4/webview_controller_delegate.dart';
+
+import 'webview_platform.dart';
+
+/// Interface for a platform implementation of a web view widget.
+abstract class WebViewWidgetDelegate extends PlatformInterface {
+  /// Creates a new [WebViewWidgetDelegate]
+  factory WebViewWidgetDelegate({
```

Comment:
Can you make this take a `WebViewWidgetCreationParams` class instead? I know Android has an additional parameter that isn't used for iOS.

Reply:
Moved the parameters into the `WebViewWidgetCreationParams` class and made sure this is passed on to the `WebViewWidgetDelegate` class.